manifests/params.pp

```diff
@@ -6,7 +6,11 @@
   $graphite_host = '127.0.0.1'
   $graphite_port = 2003
 
-  if $::is_pe {
+  $graphite_prefix = '' # lint:ignore:empty_string_assignment
```

Comment:
There's a reason the lint checks '', on puppet 4 this will cause a compile failure.  I think these need to be handled differently.